model/src/pyrenew/latent/infections.py

```diff
@@ -48,23 +49,22 @@ class Infections(RandomVariable):
 
     def __init__(
         self,
-        infections_mean_varname: str = "latent_infections",
+        latent_infections_varname: str,
     ) -> None:
         """
         Default constructor for Infections class.
 
         Parameters
         ----------
-        infections_mean_varname : str, optional
+        latent_infections_varname : str
             Name to be assigned to the deterministic variable in the model.
-            Defaults to "latent_infections".
 
         Returns
         -------
         None
         """
 
-        self.infections_mean_varname = infections_mean_varname
+        self.latent_infections_varname = latent_infections_varname
```

Comment:
Just realized that this is unused @damonbayer. Do we need to use `npro.deterministic` here?

Reply:
actually, we are already saving all_latent_infections in the model classes, so no reason to record them here as well.